kubernetes/loculus/values.yaml

```diff
@@ -1212,10 +1213,8 @@ defaultOrganisms:
     <<: *defaultOrganismConfig
     preprocessing:
       - <<: *preprocessing
-        configFile:
-          <<: *preprocessingConfigFile
-          nextclade_dataset_server: https://raw.githubusercontent.com/nextstrain/nextclade_data/ebola/data_output
-          nextclade_dataset_name: nextstrain/ebola/sudan
+      - <<: *preprocessing # test having 2 simultaneous versions
+        version: "2"
```

Comment:
@corneliusroemer, this is breaking the test dummy organism because the lineage system definitions is only provided for version 1. This is causing the SILO import job to throw "No URL defined for pipeline version 2."